cql3/statements/list_users_statement.cc

```diff
@@ -74,14 +73,14 @@ cql3::statements::list_users_statement::execute(query_processor& qp, service::qu
     const auto& cs = state.get_client_state();
     const auto& as = *cs.get_auth_service();
 
-    return auth::has_superuser(as, *cs.user()).then([&cs, &as](bool has_superuser) {
+    return auth::has_superuser(as, *cs.user()).then([&cs, &as, make_results = std::move(make_results)](bool has_superuser) {
         if (has_superuser) {
-            return as.underlying_role_manager().query_all().then([&as](std::unordered_set<sstring> roles) {
+            return as.underlying_role_manager().query_all().then([&as, make_results = std::move(make_results)](std::unordered_set<sstring> roles) {
```

Comment:
These moves are actually copies because the surrounding lambda is not `mutable`.

Reply:
yes, for now I don't want to corutinize cql3/statements code.

>These moves are actually copies because the surrounding lambda is not mutable.

ok, will fix